README.md

```diff
@@ -125,23 +119,17 @@ A list of companies that have paid Developer Community Writer Programs.
   > Technical content and tutorials about DevSecOps, Cloud security, Kubernetes security.
 
 - [Magic](https://magic-fortmatic.typeform.com/to/Wgzsocor) - Up to $300 per piece
-  [Link to Magic Website](https://magic.link/)
-  > Technical tutorials on how to use Magic
+  > Technical tutorials on how to use Magic.link
 
 - [Make Use Of](https://www.makeuseof.com/contributor/) - $120 per piece with performance benefits
   > Tutorials and features about consumer apps and software products.
 
-- [Mixster](https://mixstersite.wordpress.com/2019/05/24/mixster/#more-2253)
```

Comment:
This one is really confusing. It seems like maybe they're looking for writers, but there's just a Twitter thread, not a clear way to apply.

Reply:
true, might be worth removing for now